lib/fastly_nsq/tls_options.rb

```diff
@@ -1,15 +1,18 @@
 module FastlyNsq
-  class SSLContext
+  class TlsOptions
     def initialize(context = nil)
       @context = context || {}
     end
 
     def to_h
       merge_contexts
       if empty_context?
```

Comment:
I didn't think `empty_context?` would work anymore since you delete all the nil key's in `merge_context`. But I was wrong:

```
2.3.1 :001 > {}.all?{|k,v| v.nil?}
 => true
```

still, this might be better off as

```
@context.empty?
```